code/scripts/characters.js

```diff
@@ -0,0 +1,220 @@
+// Array with all the characters, as objects
```

Comment:
Nice that you tried to export your data to the external file 🌈 why didn't you use it in your script? 🤓

Reply:
I can't make it work, don't connect:(